1-callbacks/solution.js

```diff
@@ -32,16 +32,32 @@ node solution.js name1 name2 name3
 ** give a look to node.js util.promisify, avoid to alter the validate-user.file **
 */
 
-function solution() {
-    // YOUR SOLUTION GOES HERE
+const validateUser = require("./validate-user");
 
-    // you get your 5 names here
+function solution() {
+    const names = ["Juanito", "Pablito", "Mario", "John", "Richard"];
+    //const names = process.argv.slice(2);
+
+    let errorLog = "";
+    let successLog = "";
+    let processed = 0;
+
+    const cb = (error, user) => {
+        if (error) {
+            errorLog += error.message + "\n";
+        } else {
+            successLog += `id:${user.id}\nname:${user.name}\n\n`;
+        }
+        processed++;
+
+        if (processed === names.length) {
+            console.log("Success\n\n", successLog);
+            console.log("Failure\n\n", errorLog);
+        }
+    };
 
     // iterate the names array and validate them with the method
-
-    // log the final result
+    names.forEach((current) => {validateUser(current, cb);});
```

Comment:
You can simplify this expression by removing the brackets and semi colon in the arrow function.
```suggestion
    names.forEach((name) => validateUser(name, cb));
```

Also, consider using more meaningful names also here. To be clear, `current` is not wrong, but if you have to do a lot of work in a `forEach` callback, `current`  does not provide enough context of the value at the iteration